docs/installation/binaries.md

```diff
@@ -188,21 +235,15 @@ need to add `sudo` to all the client commands.
 > The *docker* group (or the group specified with `-G`) is root-equivalent;
 > see [*Docker Daemon Attack Surface*](../security/security.md#docker-daemon-attack-surface) details.
 
-## Upgrades
+## Upgrade Engine
 
-To upgrade your manual installation of Docker, first kill the docker
+To upgrade your manual installation of Engine on Linux, first kill the docker
 daemon:
 
     $ killall docker
 
-Then follow the regular installation steps.
-
-## Run your first container!
```

Comment:
Since this document is for "hackers", I considered the target audience "experienced" users; showing how to run a container didn't seem to fit that audience (and we already link to the user guide at the end)